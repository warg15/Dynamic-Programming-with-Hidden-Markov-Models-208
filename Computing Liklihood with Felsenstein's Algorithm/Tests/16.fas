>Pigeon
TAGTCCGCTATCTCCGTTACAATGTGTTAGCTCAGGAAACGAAAGACCGATATAAAGGCACAAGGCTATCACCATAAGTTATTTAGGAACCGTTAGGAGAGTGAATGCATAGACCCTTTTACTTAGCAAAATCTAGACCTTCTTGTTTTAAACGAGGGTCACTTGCTTTAGTAACGCAGAGATGTTAATTCTGGGCCGTT
>Chimney-swift
TAGTCCGCTATCACCGTCGGGTTGTGTTACCTCAGGAAAAGAAAGCCCGCTACAACGACATAAGTCTATCACAATAATTTATCTAGGAACTTTTTGGAGAGTGGATGCATAGACCCTCTTACTTCGCAAAGTCTAGGCCTTCTGGTATTAAATGAGGGCTATTTGTTTTAGTAACTCAAAGACATCAATTCCGGGCCGTT
>Yellow-throated-sandgrouse
TAGTCCGCTATCGCCGTCGGGTACTGTTAGCTCAAGAAATGATAGAGTGATATAAAGACATAAGTATATCACGATAAGTTATCTAGCAACTTTTAGGACAGTGGATGCATAGACCCTCTTACTCAGCAAAGTTTAGACCTTCTGATTTTAAATGAAGGTTATTTGTTTTAGTAACGCATAGACATTTATTCTGAGCCGTT
>Red-crested-turaco
TAGTCCGTTATCGCCATTGGGTAGTGTTAGCTCAGGAAGCGAAAGACCGATATAAAGACCTAAGTCTATCACGATAAGTTATCTAGGAACTTTTAAGATAGTGTATGCATAGACCCTCCTACTTAGCGAAATCTAGACCTTCTGGTATTAAATGGGTGTTATTTGTATTAGTAACCCAAAGACGTTAATTCTGGGTCGTT
>MacQueens-bustard
TAGTTCGCTGTCGGCGTCGGGTAGTGTTAGCTCAGGAAACGAGAGACTGATATAAAGACATAAGTCTATCACGATAAGTTATCTAGGAACTTTTTTGAGAGTGGATGCATAGAACCTCTTACTTAGCAAAGTCTAGACCTTCTGGTATTAAATGAGGGTTGTTTGTTTCAGTAACCGAAAGACGTTAATTCTGGGCCCTT
>Emperor-penguin
TAGTCCGCAATCGCCGCTGGGTAGTGTTAGCTCAGGAAACCAAAGACCGATATAAAGATATAAGTCTATCACGATAACTTATCTAGGAACTTTGAGGTGAGTGGGCGCATAGACCCTCTTACTTATCAAAGTCAAGACCTTCTGGTATTAAATGAAGGTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTGGGCCGTT
>Great-crested-grebe
TGGTCGGCTATTGCCTTTAGGTAGTGTTAGCTCAGGAAACGAAAGACTGATATAAAGACATAAGTCTATCACGACAAGTTATCTAGGAACTTTTAGGAGAGGGGATGTATAGAGCCTCTTACTTAGCAAGGTTTAGACCTTCCGGTATTGAATGAGGGTTATTTGCTTTAGTAACCCAAAGACGTTACTTCTGGGCCGCC
>American-flamingo
TAGTCGGCTATCGCCTTTGGGTAGTGTTAGCTCAGGAAACGAAAGGCCGATATAAAGACATAGGTCTATCACGATAAGTTATCTAGGAACTTTTAGGAGAGTGGATGCCTAGACCATCTTACTTAGCAAAGTCTAGACCTTCTGGTATTAAATGAGGGTTATTTGTTTTAGTAACCCAAAGACGGTAATTTTGGGCCGCT
>Adelie-penguin
TAGTCCGCAATCGCCGTTTGGTAGTGTTAGCTCAGGAAACCAAAGACCGATATAAAGATATAAGTCTATCACGATAACTTATCTAGGAACTTTGAGGTGAGTGGGTGCATAGACCCTCTTACTTATCAAAGTCAAGACCTGCTGGTATTAAATGAAGGTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTGGGCCGTT
>Turkey-vulture
CAGTCCGTTATCGCCGTTGGGTAGTGTTAGCTCAGGAAACGAAGGACCGATATAAAGACATAAGTCTATCACGACAAATTATCTAGGAACTTTTAGGCGACTGGATGCATAGACCCTCTTACTTAGCAAAGTCTAGACGTTCTGGTATTAAATGAGGGTTATCTGTTTTAGTAACCCAAGGACGTTAATTCTGAGCCGTT
>Hoatzin
TGGTCCGCTATCGCCGTTGGGTAGTGTTAGCTCAGGAAACGAAAGACCGATATAAAGACATGAGTCTATCACGATAAGTTATCTAAGAACTTTTAGGAAACTGGATGCATAGACCCTCTTACTTGACAAAGTCTAGACCTTCCAGTATTAAATGAGGGTTATTTGTTTTAGTAACCCAACGACGTTAATTCTGGGCCGTT
>Sunbittern
TAGTCCTATATCGCCGTTGGGTAGTGTTAGCTTAGGAAGCGATAGACCGATATAAATACAGAAGTCTATCACGATGAATTATCTGGAAACTTTTAGGAGAGTGGATGCATAGACCCTCTTACTTAGCAATGTCTAGACCTTCTGGTATTAAATGAGGGTCATTGGTTTTAGTAACCCAAAGACATTAGTCCTGGACCGTT
>Medium-ground-finch
CAGTCCGTTATCGCCATTGAGTAGTGTTAGCGCAGGAAACGAAAGACCGATCTAAAGACATAAGTGTATCACGATAAGTTACCTAGGAACTTTTAGGAAACTGGATGCCTAGACCCCCTTACTTAGGAAATTCTTAAACTTCTGGTATTAAATGAGGGTTATTTGTTTTAGTATCCTAAGGACGTTGATTCTGGGCCGTT
>White-tailed-eagle
TAGTCCGTTATCGCCGTTGTGTAGTGTGAGCTCAGGAAACGAAAGACCGATATAAAGACATAAGTCTGTTACGAGAAGTTATCTAGGAACTTTTAGGAGACTGGATGCATAGACCCTCTTATTTAGCAAAGTCTAGACCTTCTGGTATTAAATGAGGGTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTGGGCCGTT
>Peregrine-falcon
TAGTCCGTTATCGCCGTTGGGTTGTGTTATCTCAGGAAGCGAATGACTGATATAAAGGCATAAGTCTATCACAATAAGTTATCTAGGAACTTTAAGGAGACAGGATGCATAGACCCTCTTACTTAGCAAAGTCCAGACCTTCAGGCATTAAATGAGGGTTATCTGTTTTAGTAACCCACAGACATTAATTCTGGGCCGTC
>Little-egret
TAATCCGCTATCGCCGTTGGGTAGTGTTGGCTCACGAAACGAAAAACCGATATAAAGACATAAATATATCACGATAAGTTATCTAGGAACTTTTAGTAGAGTGGATGCATGGACTCTCTTACTTAGCAAAGTCTAGAGTTTCTGGTATTAAATGAAGGTTATTTGTTTTAGTAACCCAAAGACGTTAATGCCGGGCCGTT
>Chicken
TAATCCGCCATCGCCTTTGGGTATTGTTACATAAGGGAGCGAAAGACCGGAATAAAGACATAAGTTTATCACGAGAGGTTACCTAGGAATTTTTAGCAGAGTGGGTACTTAGACCCTCCTGCTTACCAAAGTCTAGACCTTCTGGCATTAAAGGAGGTTTATTTGTTTTAGTAACCAAGAGACGTTAATTTTGGGACGTT
>Chuck-wills-widow
TAGTCCGGTATCCCCGTCGGGTAGTGTTAGCTCAGGTAACGAAAGACCGACATAAAGGCATAAGTCTATCACCATAAGTTATCTAGGAACTTTTAAAAGAGTGGATGCATAGACCCTGTTACATAACAAAATCTAGACCTTCTGGTATCGAATGAGGGTTATTTGTTCTAGTAACCCAAAGACGTTAATTCTAGGCCGTT
>Rifleman
CAGTCCGTTTTCGCCGTTGGGTAGTGTTAGCGCAGGAAACGAAAAACTGATATAAAGACATAAGTCTATCATAATAAGTTATCTAGGAACTTTTAGGAGACCGGATGCATAGACGCTCCTACTTCGGAACAGCCAAACCTTCTGGTATTAAATGAGGGTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTAGGCCGTT
>White-throated-tinamou
TAGCACGCTACCGTCGTTGGGTAGTGTTAGGTCAGGAAACGGAAGACCGACATAAATACATAGGTTTATCACGGTAAGTCATATACGAACTTTTAGAAGAACGGATCCATTGGCTCTTTTGTTTAGCAAAGTATAGACCTGCAAGAATTAAACTGGGGTTATTTCTATGAGTAACCCATAGATATTAATTTTGCGACATT
>Annas-hummingbird
TAGTTCCCTATCATCGTTGGGAAGTGTTAGCTCAGGATACGAACGCCCGCTAGAAAGACATAATTCTATCGCGATAAGCTACCTAGGAACTTTTCGGAGAGTGGATGCATAAACCCTCTTACTTAGCAGAATCTATACCTCGGGGTATTACATGAGGGTTATTTGTTTTTGGAACCCAGAGACATTAATTCTAGGGGGTT
>Red-legged-seriema
TAGTCCGCTATCGCTGCTGGGTAACGTTAGCTCAGGAAACGAAAGATCGATATAGAGACATAAGGCTATCACGATCAGTTATCTAGGAACTTTTAGGAGACGGGATGCATAGACCCTCTTACTTAGCAAATTCTAGACCTTCTGATATTGAATGAGGGTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTGGGCCATT
>Bar-tailed-trogon
TAGTCCATTATCGCCGTCGGGTAGTGTTAGCTCAGGAAATGAAAGACCGATATAAAGACATAAGTCTATCACGAGAGGTTACCTAGGAACCTTTGGGAGACTAGATGCGTAGACCCTCTTACTTAACTAAATCTAAACCTTCTGGTATTAAATGAAGGTTATTCGTTTTAGTAACCCAAAGACATTAATTCTGGGCCGTT
>Common-ostrich
TAGCCCGCTACCGCCGTTGGGTATTGTTAGCTCAGGAGACGGAAGACCGATAAGAAGATATAAGTTTATCATGGTAAGTTAAATATGTACTTTTAGGAGAGCGGATCCATAGGCTCTCTTGTTTAGCAAAATATAGACCTTCCGGTATTAAATGGGGGTTATTTCTTTGAGTAACCCAAAGATGTTAATTCAGTGACATT
>Zebra-finch
CAGTCCGTAATCGCAGTTGGGTAGTGTTAGCGCAGGAAACGAAAGACCGATATAAAGACATAAGTGTATCGCGATAAGTTACCTAGGAACTTTTAGGAGACTGGATGCATAGACCCTCTTACTTAGAAAAATCTTAAACCTCTGGTATTAAATGAAGGTTATTTGTTTTAGTATCCTAAAGACGTTGATTCTGGACCGTT
>Northern-fulmar
TAGTCCGCTATCGCTGTTGGGTAGTGTTAGCTCAGCAAACGAAAGACCGATATAAAGACATAAGTCTATCACGATAAGTTATCTAGGAACTTTTAAGAGAGTGGATGCATAGACCCTCTTACTTAACAAAGTCTAGACCTTCTGGTATTAAATGAAGCTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTGGGCCGTT
>Kea
CAGTCCGTTATCGCCGTTGGGTAGTGTTAGTTCAGGAAGCGAATGACCGATATTAAGACATAAGTCTATCACGATAAGTTATCTAGGAATTATTAGGAGACTGGATGCATAGACCCTCTTACTTAGCAAAGTCTAGACCTTCTGGTATTGAATGAGGGTTATTTATTTTAGTCACCCAGAGACCTTTATTCTGGGCTGTT
>Grey-crowned-crane
TACTCCGCGATCGCGGTTGGGTAGTGTTAGCTCAGGAAACGAAAGACCGATACAAAGACATAAGTCTATCACGAGAAGTTATCAAGGAACTTTTAGGAGAGTGGATGCAAGGACCCTCTTACTTAGCAAAGTCTAGACCTTCTGGTATTAAATGAGGGTTATTTGTTTTAGTAACTCAAAGATGTTAATTCTTAGCCGCT
>White-tailed-tropicbird
TAGTGCGCTATCGCCGTTGGGTACTGTTAGCTCAGGAAACGATAGACCGATATAAAGACATAAGTCTATCACGATAAGTTATCTGGAAACTTTCAGGCGAGTGGATGCATAGACCCTCTTACTTAGCAAAGTCTAGACCTCCTGGTATTAAATGAGGGTTATTTGTTTTAGTAACCCAAAGACGTTAATGCTGGGCCGTT
>Red-throated-loon
TAGTCCGCTCTCGCCGTCGGGTAGTGTTAGCTCAGGAGACGAAAGACCGATATAAAGACATAAGTCTATCACGATAAGTTCTCAAGGAACTTTTAGGAGAGCGGACGCATAGACCCTCTTACTTAGCAAAGCCTAGACCTTCTGGTATTAAATGAAGGTTATTTGTTTTGATAACCCAATGACGTTAATTCTGGGCCGTT
>Golden-collared-manakin
CAGTCCGTTATCGCCGTCGGGTAGTGTTAGCGCAGGAAACGAAAAACCGATATAAAGACATACGCGTATCACGATAAGTTATCTAGAAACTTTTAGGAGACTGGATGTATAGACCCTCTTACTTAGAAAAGTCTAAAATTTCTGGCATTAAATGGGGGTCATGTGTTCTAGTAACCCAAAAACGTTAATTCTGAGCCGTT
>Speckled-mousebird
TAGTCCGTTATCGGCGTTGGGTAGTCTGAGACCAGGTCGCAAAAGACTGTCCTAAAGGCATAAGTCCGTCACGCTATGTTATCTAGGAACTTTGAGGAGACTGGGTGCGCAGATTCTCTTACTTAGCAATGTCTAGACCTTCTGATATTAAATAAGAGGTATGTATCTTAATAACCTAAAGACGCTAATTCTGGGCTGTT
>Brown-mesite
TAGTCCGATATCGCCGTTGGGTAGTGTTAGTTCAGGAAACTAAAGACCGATATAAAGATATAAGTCTATCACTATAAGTTATCTAGGAACTTCTAGGACAATGGATGCATAGACCCTCTTACTTAGCAAAGTCTAGCCCTTCTGGTTTTAAATGAGGGCTATTTGTTTTAGAAATGCAAAGACGTTAAATCTGGGCCGTT
>Budgerigar
CAGTCCGTTATCGCCGTTGGGTAGTGTTAGCTCAGGAAGCGTATGACCGATATAAACACACAAGTCTATCACGATAAGTTATCTGGGAATTATTAGGAGACTGGATGCACAAACCCTCTTACTTAACAAAGTCTAGACCTTCTGGTATTGAATGAGGATTATTTGTGTTAGTAACCCAAAGACCTTAATTCTGGGCCGTT
>Killdeer
TAGCCCTATATCGCCGTTGGGTAGTCTTAGCTCAGGAAACGAAAGACCGATACAGAGACCTAAGTCTATCACGATAAGTTATCTAGGAACTTTTAGGAGAGTGGATGCAAAGGCCCTCTTACTTAGCAAAGTCTAGACCTTCTGGTATTAAATAGGGGTTATTTGTTTTAGTACCCCAAAGACGTTAGTTCTGGGCCGTT
>Carmine-bee-eater
TAGTCCGTTATCGCCTTTGGGTAGTGTTAGCTCGGGAAACGAAAGAGCGAGCTAAAGACATATGTCTATCACGATAAGTTATCTAGGAACTTGTAGGAGACTGGATTCAGAGACCTTCTTACTTAGCAAAGTCTAGACCTTCTGGTATTAAATAAGGGTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTGGGCCGTT
>Downy-woodpecker
TAATCCGTTATCGCTGTTGGACGGTGCTAGCTTGGGAGACAGGAATCCGATAGAAAGACATAAGTCTATCACGACAAATTGTCTAGGAATTTTTAGGAGACTGGACGCATAGACCATCTTACATAGCAAAGTCTACACTTTTTAGTATTAAATGAGGGTTAATTGTCTTAGTAAAGCAAAGACATTTTTTCTGGGCCGTT
>Great-cormorant
TAGTCCGCTATCGCCGTTGGGTAGTGTTGGCTCAGGAAGCGAAAGACTGATATCAAGACATATGTCTATTACGATAAGTTATCTAGGAACTTTTAGGAAGGTTGATGCATAGACCCTCTTACTTATCAAAGTCTAGACCTTCTGGTATTAAATGAAGGTTATTTGTCTTAATAACCCGAAGATATTAATTCTGGGACGTT
>American-crow
CAGTCCGTTATCGCCGTTGGGTAGTGTTAGCGCAGGAAACGAAAGACCGATATAAAGACATAAGTGTATCACGATGAGTTATTTAGGAACTTTTAGGAGACTGGATGCACAGACCCCCTTACTTGGGAAAGTTTTAAACTTCTGATATTAAATGAGGGTTATTTGTTTTATTAACCCAAAGACGTTGATTCTGGGCCGTT
>Crested-ibis
TAGTCCGCTATCGCCGTTGGGTAGTGTTGGCTCAGGAAACGGAAGACCGATATAAATACATAAGTCTACCATGATAAGTTACCTAGGAACTTTTAGGAGAGTGGATGCATAGACCCCCTTACTTAGCAAAGTCTAGACCTTCTGGTATTAAATGAAGGTTATTTGTTTTAGTAAACCAAAGACGTTAATTCTGGGTCGTT
>Cuckoo-roller
TAGTCCTTTATCGCCGTTGGGTAGTGTTAGCTCAGGAAACGAAAGACCGATATCAAGACATAAGTCTATCACGTTAAGTTATCTAGGAACTTTTAGGAGACTGCATACAAAGACCGTCTTGATTGGAAAAGTCTAGACCTTCTGGTATTAAATGAAAGTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTGGGCCGTT
>Common-cuckoo
TAGTCCGCTATCGCCGTTGGGTAGTATTAGCTCAGGAAACGAAAGACCGATATAAAGGCATAAGTCTATCACGGTAAGTTATCTAGGAACTTTTAGGAGAGTGGATGCATAGACCCTCTTGGTTGGCGGAGTTTAGGCTTTCTGTTATTAAATAAGGGTTATGTGTTTTAGTAACCAAAAGACGTTAATTCTGGACCGTT
>Barn-owl
TAGTCAGTTATCGCCGTCGGGTAGTGTTAGCTCAAGTAATGAAAGACCGATATAAAGATATAAGTCTATCACGATAAGTTATCTAGGAACTTTTAGCAGACTGGATACATAGACTCTCTTACTTAGCAAAGTCTAGTCTTTCTGGTAGTAAATGAGATTTATTTGTTTTTGTGACCCGAAGACGTTCACTCTGGGCCGTT
>Bald-eagle
TAGTCCGTTATCGCCGTTGTGTAGTGTGAGCTCAGGAAACGAAAGACCGATATAAAGACATAAGTCTGTTACGAGAAGTTATCTAGGAACTTTTAGGAGACTGGATGCATAGACCCTCTTATTTAGCAAAGTCTAGACCTTCTGGTATTAAATGAGGGTTATTTGTTTTAGTAACCCAAAGACGTTAATTCTGGGCCGTT
>Pekin-duck
TAGTCCACTATCGTCGTTGGGTAATGTTCGCTCAGGAAACGAGGGACCGACATAGAGCCATAAGTCTGTCACGATAAGTTACCTAGGAACTTTTAGGAGGGTGGATTCACAGACCCTCTTGCTTGGCAAAGTCTAGGCCTTCTGGTATTAGAGGATGGTTATTTATTTTAGTAACCCGAAGACGTTAATTCTGGGTCGTT
>Turkey
TAATCCGCCATCGCCTTTGGGTATTGTTACATAAGGGAGCGAAAGACCGGAATAAAGACATAAGTTTATCACGATAAGTTACCTAGGAATTTTTAGCAGAGTGGGTACTTAGACCCTCCTGCTTACCAAAGTCTAGACCTTCTGGCATCAAACGAGGTTTATTTGTTTTAGTAACCAAGAGACGTCAATTTTGGGACGTT
>Rhinoceros-hornbill
TAGTCCGTTATCGCCATCGGGTAGTGTTAGCTCAGGATACGAAAGACCGATACCGAGACATTAATCCATCACAATAAGTTCTTTAGGAACTTTTAGGAGCCTGGATACATAGACCCTCTTACTTAGCAAATTCTAGACCTTCTGGTATTAAATGAGCTTTATTTGTATTAGTAACCCAAAGACCCTAATTCTGGGCCGTT
>Dalmatian-pelican
TAGTCGGCTATCGCCGTTGGGTAGTGTTGGCTCAGGAAACGACAGACTGATATAAAGACATAACTCTGTCACGATAAGTTATCTAGGAACTTTTAGGAGAGTGGATGCATAGACCCTCTTACTTAGCAAAGCCTAGACCTTCTGGTATTAAATGAAGGTTATTTGTTTTAGTAACCCAAATACGTTAATTCTGGGCCGTT
