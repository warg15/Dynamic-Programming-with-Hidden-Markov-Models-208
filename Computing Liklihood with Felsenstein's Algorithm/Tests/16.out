-3519.835036703846
